torch/csrc/Module.cpp

```diff
@@ -1132,6 +1133,7 @@ PyObject* initModule() {
   torch::jit::initJITBindings(module);
   torch::monitor::initMonitorBindings(module);
   torch::impl::dispatch::initDispatchBindings(module);
+  torch::dynamo::initDynamoBindings(module);
```

Comment:
Isn't this going to conflict with the dynamo that is provided by the `torchdynamo` package?

Reply:
The torchdynamo package is going away.

Reply:
What about the transition time during which the version in core is "disabled" and `torchdynamo` remains the source of truth?

Reply:
https://github.com/pytorch/torchdynamo/pull/1538 makes top level torchdynamo use `torch._C._dynamo` for the C++ bits.